Gu.Wpf.PropertyGrid.NumericRows/NumericRow{T}.cs

```diff
@@ -55,6 +59,49 @@ string INumericFormatter.Format(IFormattable value)
             return value.ToString(string.Empty, culture);
         }
 
+        public override void OnApplyTemplate()
+        {
+            base.OnApplyTemplate();
+
+            var valueBinding = BindingOperations.GetBindingExpression(this, ValueProperty);
+            if (valueBinding?.ParentBinding.UpdateSourceTrigger == UpdateSourceTrigger.PropertyChanged)
+            {
+                foreach (var dependencyObject in this.RecursiveChildren())
+                {
+                    BindingExpression bindingExpression = null;
+                    switch (dependencyObject)
+                    {
+                        case DecimalBox decimalBox:
+                            bindingExpression = decimalBox.GetBindingExpression(DecimalBox.ValueProperty);
+                            break;
+                        case DoubleBox doubleBox:
+                            bindingExpression = doubleBox.GetBindingExpression(DoubleBox.ValueProperty);
+                            break;
+                        case FloatBox floatBox:
+                            bindingExpression = floatBox.GetBindingExpression(DecimalBox.ValueProperty);
+                            break;
+                        case IntBox intBox:
+                            bindingExpression = intBox.GetBindingExpression(FloatBox.ValueProperty);
+                            break;
+                        case LongBox longBox:
+                            bindingExpression = longBox.GetBindingExpression(LongBox.ValueProperty);
+                            break;
+                        case ShortBox shortBox:
+                            bindingExpression = shortBox.GetBindingExpression(ShortBox.ValueProperty);
+                            break;
+                    }
```

Comment:
We should probably check for textbox here as we have no control over what is used in the template.